application/backend/src/main/resources/apps/bulma/components/table/.content.json

```diff
@@ -0,0 +1,11 @@
+{
+  "isContainer": true,
+  "allowedComponents": [
+    "/apps/bulma/components/table/tablehead",
+    "/apps/bulma/components/table/tablefoot",
+    "/apps/bulma/components/table/tablebody"
```

Comment:
It should have also the possibility to add tablerow.

Reply:
Fixed!